app/controllers/marine_controller.rb

```diff
@@ -145,21 +144,8 @@ def designations
     end
   end
 
-  def total_coverage
-    @totalMarineProtectedAreas = 15357
-    @oceanProtectedAreasPercent = 6.40
-    @oceanProtectedAreasKm = 23183060
-  end
-
-  def distributions
-    @distributions = {
-      nationalWaters: 39,
-      nationalWatersPa: 16.02,
-      nationalWatersKm: 22624944,
-      highSeas: 61,
-      highSeasPa: 0.25,
-      highSeasKm: 558116
-    }
+  def marine_statistics
+    @marine_statistics = Wdpa::MarineStatsImporter.stats
```

Comment:
Since we are importing data into Redis, there is no need to call the importer in the controller.
You can still add this importer to the list of importers called in the `wdpa/importer.rb` file.
So the only thing left to do in the controller is retrieving those data from Redis